### PR TITLE
Benjamin

### PR DESCRIPTION
**Pull request by timo637**

### CODE REVIEW

**Comment by dejanstrancar** [comment]
Hey @timo637! Good stuff!

Just few notices:
When you are naming branches name them by feature not as this one "Benjamin".
Also when you are adding comments into code add them in English. In general everything about development environments is usually in English, because you dont know who will help you around the code in future phases.

Also i would suggest you to write code like this:

```
if (crouching){
   transform.localScale = new Vector3(1, 1, 1);
} else {
    transform.localScale = new Vector3(1, 2, 1);
}
```

instead of

```
if (crouching)
{
    transform.localScale = new Vector3(1, 1, 1);
}
else
{
    transform.localScale = new Vector3(1, 2, 1);
}
```

see code length difference? Much less space take for same functionality.

Anyway keep up with good code!


P.S: Govorim Slovensko, ampak se v development svetu govori v Anglescini, tako da upam da te ne moti ce dodajam komentarje v Anglescini ;)

**Comment by timo637** [comment]
Hello Dejan,

thank you for reading the code and making some very useful comments. You are completely right and I will translate the comments immediately. About the code form suggestion, I agree that it is too bloated, but it is the editor that we use - visual studio - that automatically puts it in such form, so we just went with it.

Anyway, thanks again and hope you're well!:)

**Comment by dejanstrancar** [comment]
Ur using Visual Studio or Visual Studio Code?

**Comment by timo637** [comment]
The first one. Though I think Benjamin does actually use VSC.

File: placeholderName/Assets/Scripts/Player.cs [hunk 1]
```diff
@@ -18,27 +21,61 @@ void Start()
...
+        if (Input.GetKey("s"))
+        {
+            crouching = true;
+            characterVelocity = 2;
+        }
+        else
+        {
+            crouching = false;
+
+            if (Input.GetKey("left shift") && touchingGround)
```
Comment (dejanstrancar) [inline placeholderName/Assets/Scripts/Player.cs:43]: That if could go out from GetKey('s') statement.

So GetKey('s') is only for crouching.

**Review by dejanstrancar — APPROVED** [review]

**Comment by dejanstrancar** [comment]
@timo637 @Anej9 is that ok that it was closed and not merged? Closing without merging means changes were not added to main code.

**Pull request closed without merging**